examples/plot_firing_pattern.py

```diff
@@ -16,6 +16,7 @@
 
 import hnn_core
 from hnn_core import read_params, Network
+from hnn_core.neuron import _neuron_network
```

Comment:
private functions should note be exposed in examples. The idea is that you want to keep most functions/classes private and only expose very few in public. If you do this, the internal architecture can be changed in the future without impacting the user

Reply:
Let's talk about this example. I wasn't aware of it until I tried building the docs. My intention was not to directly expose these functions.

I suggest we remove this example for now. In the future, accessing cells would be done through Network. We should think carefully about that interface though.